app/bot.py

```diff
@@ -21,22 +21,34 @@ def send_message_to_chat(self, chat_id: str, exchange: Exchange, img_path: str =
         bot: telegram.Bot = telegram.Bot(token=self.token)
         logging.info("Calling Dashboard callbacks")
         
-        if exchange != Exchange.BITKUB:
+        if exchange == Exchange.BITKUB:
```

Comment:
try this ? https://www.python.org/dev/peps/pep-0636/

Reply:
This feature is for `python 3.10` and it might be unstable. I'll keep it this way